manifests/charts/global.yaml

```diff
@@ -36,9 +36,11 @@ global:
   # Enabled by default in master for maximising testing.
   istiod:
     enableAnalysis: false
-
+  # `centralIstiod` flag will be deprecated in favor of `externalIstiod` to better support external control plane model in 1.9.
```

Comment:
1.9?  I recommend removing versions... version should be 1.8 here.

Reply:
fixed

Reply:
removed version